pytest/tests/mocknet/outage.py

```diff
@@ -11,14 +11,29 @@
 
 nodes = mocknet.get_nodes()
 
-validator_accounts = mocknet.list_validators(nodes[0])
-# Nodes 0, 1, 2, 3 are initially validators
-assert validator_accounts == {'node0', 'node1', 'node2', 'node3'}
+# Get the list of current validators, sorted by node index
+validators = sorted(nodes[0].get_validators()['result']['current_validators'],
+                    key=lambda v: int(v['account_id'][4:]))
+assert len(validators) == (mocknet.NUM_NODES // 2)
```

Comment:
is there any reason why only half of the nodes are validators?

Reply:
This was always true on mocknet. I'm not sure if it was a decision made my @ailisp or by @SkidanovAlex but I didn't change it when I scaled up to 50 nodes.

Reply:
inherited from devnet where 4 validator, 4 non validtor node. for mocknet feel free to use any numbers.

Reply:
Sure, I'm happy to change in whatever way is desired. Just let me know @SkidanovAlex and @bowenwang1996 what you think is best.

Reply:
I think having a small (< 10) number of rpc nodes should be sufficient.